src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketBuildStatusNotifications.java

```diff
@@ -143,11 +155,20 @@ private static void createStatus(@NonNull Run<?, ?> build, @NonNull TaskListener
 
     private static void sendNotifications(Run<?, ?> build, TaskListener listener)
             throws IOException, InterruptedException {
-        final SCMSource s = SCMSource.SourceByItem.findSource(build.getParent());
+        SCMSource s = SCMSource.SourceByItem.findSource(build.getParent());
```

Comment:
This line will fail if the job already got "disabled" (e.g. due to not building feature branches that are also build in a Pull Request).
The job will forever be stuck in Bitbucket as "In Progress" and the user will never be able to merge it.